### ISSUE TITLE
add renku logo for renku-based workflows

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/frontend-astrooda#121]
![image](https://user-images.githubusercontent.com/3909535/152138084-f13616a2-43c3-4319-bbc1-d72396b66901.png)

**Comment by volodymyrss** [comment]
@burnout87 , @motame  this does not look too complex, could it be done in the current release? 
Several, pre-defined, tabs will have this logo.

**Comment by motame** [comment]
Is it OK ?

![renku-explore](https://user-images.githubusercontent.com/44972498/161737958-8bc8d5cc-af4f-4ec0-8ee1-edeb43ea5f1c.png)

**Comment by volodymyrss** [comment]
maybe it's better to just replace `renku` word with the label 
![image](https://user-images.githubusercontent.com/3909535/161753882-6bb9aa1e-80b7-4ded-bc43-19c6ef36220f.png)

**Comment by burnout87** [comment]
> maybe it's better to just replace `renku` word with the label

I agree, alternatively there could just be the frog logo and then the label doesn't change...

**Comment by volodymyrss** [comment]
> > maybe it's better to just replace `renku` word with the label
> 
> I agree, alternatively there could just be the frog logo and then the label doesn't change...

yes, this could work

**Comment by motame** [comment]
OK ! logo without label.

![firefox_D4HngN0nxH](https://user-images.githubusercontent.com/44972498/161769167-664da335-d324-4844-b65d-e4e3b07ad953.png)

**Comment by burnout87** [comment]
Cool! I like it

**Comment by volodymyrss** [comment]
Yes, thanks, this seems to be a good way to combine the given text with the logo. Though I think the text is a bit long. Let's discuss later.

**Comment by motame** [comment]
I think that just putting the logo with the label (as it was the first time) without the text "Explore ..." should be enough as there is an info icon giving more details ...

**Comment by burnout87** [comment]
mmm You mean just the logo alone? Not a bad idea but I would still keep some text, perhaps just "Explore on Renku", and a more detailed description in the information tooltip

**Comment by volodymyrss** [comment]
Also it's not exactly "Explore" here. 
Maybe just "View on Renku" or "Open in Renku"

**Comment by motame** [comment]
I would go for "View on Renku"

**Comment by burnout87** [comment]
I think "Open in Renku" is the most appropriate here

**Comment by volodymyrss** [comment]
Between "View" and "Open" I rather prefer View, but I am also not too happy about neither. Let's discuss on a meeting. There are other issues.

**Comment by motame** [comment]
> I think "Open in Renku" is the most appropriate here
If it does open a window (a modal or new tab/window), I would go for it ...
